frontend/src/layout/header/UserBox.tsx

```diff
@@ -380,7 +487,12 @@ const Logout: React.FC = () => {
                 "error": () => <FiAlertTriangle />,
             })}
             borderTop
-            css={{ color: "var(--danger-color)" }}
+            css={{
+                color: "var(--danger-color)",
+                ":hover, :focus": {
+                    borderRadius: "0 0 8px 8px",
+                },
```

Comment:
The borderRadius can be set without hover/focus right? I mean yes, it's only needed then, but it doesn't hurt if its set all the time?

Reply:
True. I decided to move this "radius for hover/focus purposes" thing to the `FloatingMenu` now (without the pseudo classes, as you suggested), so it can be used by both menus.